controller/src/main/java/io/pravega/controller/store/stream/StoreException.java

```diff
@@ -114,12 +134,42 @@ public NodeNotEmptyException() {
     }
 
     /**
+     * Exception type when you are attempting to update a stale value.
+     */
+    public static class WriteConflictException extends StoreException {
```

Comment:
good catch, totally missed this when moving the class. Updated to use the trait now to keep functionality same as before.